packages/netlify-cms-core/src/constants/publishModes.ts

```diff
@@ -20,3 +20,4 @@ export const statusDescriptions = Map({
 });
 
 export type Status = keyof typeof Statues;
+export type StatusValues = typeof Statues[Status];
```

Comment:
Is there a good reason for changing the type of `default_workflow_status`?

Reply:
Yeah you right, I will remove this type

Reply:
I just add a last fixup to verify if the status is a valid one @bytrangle